packages/flame/lib/src/game/camera.dart

```diff
@@ -143,10 +146,10 @@ class Camera extends Projector {
   /// When using this method you are responsible for saving/restoring canvas
   /// state to avoid leakage.
   void apply(Canvas canvas) {
-    canvas.translateVector(
-      (-position)..scale(zoom),
-    );
-    canvas.scale(zoom);
+    _transform.setIdentity();
+    _transform.translate(-_position.x, -position.y);
```

Comment:
That is a good idea, in this case it would give very little I think though since it only sets doubles in an already existing matrix.
But it is such a simple improvement that I'll do it anyways and then we can follow suit on the other places where we'll use the matrix transformations.

Reply:
translate2(-_position) ?

Reply:
It would be less efficient since it has to create a new negated `Vector2` object